src/components/SiteDetail.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import summary from '../apis/summary';
+class SiteDetail extends Component {
```

Comment:
Can we get a space between the imports and the class declaration?

Reply:
yes, I missed this